lib/livebook_web/live/sidebar_helpers.ex

```diff
@@ -64,11 +81,42 @@ defmodule LivebookWeb.SidebarHelpers do
     ~H"""
     <span class="tooltip right distant" data-tooltip="User profile">
       <%= live_patch to: @path,
-            class: "text-gray-400 rounded-xl h-8 w-8 flex items-center justify-center",
+            class: "text-gray-400 rounded-xl h-8 w-8 flex items-center justify-center mt-2",
             aria_label: "user profile" do %>
         <.user_avatar user={@current_user} text_class="text-xs" />
       <% end %>
     </span>
     """
   end
+
+  ## Shared home functionality
+
+  @doc """
+  A footer shared across home, settings, explore, etc.
+
+  Note you must call `shared_home_handlers` on mount/3.
+  """
+  def shared_home_footer(assigns) do
+    ~H"""
+    <.break_item />
+    <.shutdown_item />
+    <.link_item
+      icon="settings-3-fill"
+      label="Settings"
+      path={Routes.settings_path(@socket, :page)}
+      active={false} />
+    <.user_item current_user={@current_user} path={Routes.home_path(@socket, :user)} />
```

Comment:
Note that every live page has its own user path, because we show it as a modal, so the path needs to be passed from upstream.